### ISSUE TITLE
Suggestion of new features as console commands

### ISSUE THREAD

**Issue by Quetzacoalt91** [PrestaShop/PrestaShop#12172]
**Is your feature request related to a problem? Please describe.**
Deploying a PrestaShop could be made easier by adding more feature to the `console` panel. Some needs have been reported from PS Ready, Docker users etc.

**Describe the solution you'd like**

Here is a list of nice-to-have features:
* Enabling SSL on the shop (https://github.com/PrestaShop/docker/issues/56)
* Managing employees (Add / Remove / Reset password)
* Managing Webservice feature and users
* Managing Shop domain (also with multishop)
* Modules
  * Upgrade all
  * Status

Many of them could be merged in a single command allowing the management of the configuration data.

**Comment by matks** [comment]
⚠️  **Warning**: I have tried to introduce a Command and I stopped when I found out that many, many Symfony services are not usable in a CLI context because they rely on legacy code that only works in a WEB context. Most errors were something like "missing shop ID", "missing employee ID", "missing language ID", "PS_LANG_SOMETHING not defined" ...

The PR is here https://github.com/PrestaShop/PrestaShop/pull/12127

**Comment by Quetzacoalt91** [comment]
I guess I had the same issue with the module management command, and had to init the legacy core to make it work properly.

See https://github.com/PrestaShop/PrestaShop/blob/7b87a098090620f62bb4e67040c50c0810a047e6/src/PrestaShopBundle/Command/ModuleCommand.php#L84

**Comment by matks** [comment]
@Quetzacoalt91 then shoul we init it in the `bin/console` bootstrap ?
https://github.com/PrestaShop/PrestaShop/blob/develop/bin/console#L15
or add a bootstrap file dedicated to CLI, and boot the legacy core there

**Comment by jolelievre** [comment]
yep, same problem when to generate mail templates https://github.com/jolelievre/PrestaShop/blob/412482df497bc5537c65cab9e6850a07d42126f5/src/PrestaShopBundle/Command/GenerateMailTemplatesCommand.php#L92

It's even worse than that, I had issues such as:
- hook listeners don't work if you haven't `shop` and `employee` set in the legacy context
- many services depend on the legacy `Configuration` including a twig `LayoutExtension` so it make even basic symfony command fail (example: `debug:event-dispatcher`)

This would be the occasion to clean the bundle, let's at least remove `Configuration` with an appropriate symfony service
